boards/seeed_wio_terminal.json

```diff
@@ -0,0 +1,54 @@
+{
+  "build": {
+    "arduino": {
+        "ldscript": "flash_with_bootloader.ld"
+    },
+    "core": "seeed",
+    "cpu": "cortex-m4",
+    "extra_flags":"-D__SAMD51P19A__ -DWIO_TERMINAL -DSEEED_WIO_TERMINAL -DSEEED_GROVE_UI_WIRELESS -D__SAMD51__ -D__FPU_PRESENT -DARM_MATH_CM4 -mfloat-abi=hard -mfpu=fpv4-sp-d16",
```

Comment:
Please use an array, for example, like here https://github.com/platformio/platform-atmelsam/blob/develop/boards/adafruit_feather_m4.json#L8-L17

Reply:
These flags are used floating points, they are listed in the `platform.txt`.

Reply:
But they are also specified here https://github.com/platformio/platform-atmelsam/blob/develop/builder/frameworks/arduino/arduino-samd.py#L62-L72

Reply:
Noted, will remove these flags here